book-app/src/app/pages/list/list.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="container">
+    <mat-card class="card">
+        <mat-card-title-group>
+            <mat-card-title>書籍登録</mat-card-title>
+        </mat-card-title-group>
+        <mat-card-content>
+            <form class="example-form">
+                <mat-form-field class="input" appearance="fill">
+                    <mat-label>書籍名</mat-label>
+                    <input matInput >
+                </mat-form-field>
+                <p>
+                    <mat-form-field class="input" appearance="fill">
+                        <mat-label>あらすじ</mat-label>
+                        <textarea matInput class="textarea"></textarea>
+                    </mat-form-field>
+                </p>
+                <p>
+                    <mat-form-field class="evaluation" appearance="fill">
+                        <mat-label>評価点</mat-label>
+                        <input matInput type="number" min="0" max="100">
+                    </mat-form-field>
+                </p>
+            </form>
```

Comment:
`<p>`で区切るの気持ち悪くないかな？と思ったらMaterial側でも`<p>`使ってるんですね〜
個人的には↓が気持ちいいですが任せますm(_ _)m
cssの方にもコメントしときます。
```suggestion
            <form class="book-form">
                <div class="form-container">
                    <mat-form-field class="name full-width" appearance="fill">
                        <mat-label>書籍名</mat-label>
                        <input matInput>
                    </mat-form-field>
                    <mat-form-field class="detail full-width" appearance="fill">
                        <mat-label>あらすじ</mat-label>
                        <textarea matInput></textarea>
                    </mat-form-field>
                    <mat-form-field class="evaluation half-width" appearance="fill">
                        <mat-label>評価点</mat-label>
                        <input matInput type="number" min="0" max="100">
                    </mat-form-field>
                </div>
            </form>
```